### ISSUE TITLE
Error thrown when trying to close an untitled tab [Ubuntu 14.04]

### ISSUE THREAD

**Issue by ybounya** [atom/tabs#128]
[Enter steps to reproduce below:]

1. Just trying to close a tab
2. Seem to happen everytime after upgrade
=> actually, I can close untitled tab if they are empty

**Atom Version**: 0.188.0
**System**: linux 3.13.0-46-generic
**Thrown From**: [tabs](https://github.com/atom/tabs) package, v0.67.0

### Stack Trace

Uncaught Error: /opt/atom/resources/atom.asar/browser/api/lib/dialog.js:1
(function (exports, require, module, __filename, __dirname) { .exports.DEFAULT
                                                              ^
Unexpected token .
/opt/atom/resources/atom.asar/browser/api/lib/dialog.js:1
(function (exports, require, module, __filename, __dirname) { .exports.DEFAULT
                                                              ^
SyntaxError: Unexpected token .
    at exports.runInThisContext (vm.js:75:16)
    at Module._compile (module.js:458:25)
    at Object.Module._extensions..js (module.js:493:10)
    at Module.load (module.js:370:32)
    at Function.Module._load (module.js:325:12)
    at Module.require (module.js:380:17)
    at EventEmitter.<anonymous> (/opt/atom/resources/atom.asar/browser/lib/rpc-server.js:128:79)
    at EventEmitter.emit (events.js:119:17)
    at EventEmitter.<anonymous> (/opt/atom/resources/atom.asar/browser/api/lib/web-contents.js:99:23)
    at EventEmitter.emit (events.js:119:17)

```
At /opt/atom/resources/atom.asar/renderer/api/lib/remote.js:77

Error: /opt/atom/resources/atom.asar/browser/api/lib/dialog.js:1
(function (exports, require, module, __filename, __dirname) { .exports.DEFAULT
                                                              ^
Unexpected token .
/opt/atom/resources/atom.asar/browser/api/lib/dialog.js:1
(function (exports, require, module, __filename, __dirname) { .exports.DEFAULT
                                                              ^
SyntaxError: Unexpected token .
    at exports.runInThisContext (vm.js:75:16)
    at Module._compile (module.js:458:25)
    at Object.Module._extensions..js (module.js:493:10)
    at Module.load (module.js:370:32)
    at Function.Module._load (module.js:325:12)
    at Module.require (module.js:380:17)
    at EventEmitter.<anonymous> (/opt/atom/resources/atom.asar/browser/lib/rpc-server.js:128:79)
    at EventEmitter.emit (events.js:119:17)
    at EventEmitter.<anonymous> (/opt/atom/resources/atom.asar/browser/api/lib/web-contents.js:99:23)
    at EventEmitter.emit (events.js:119:17)
    at metaToValue (/opt/atom/resources/atom.asar/renderer/api/lib/remote.js:77:15)
    at Object.exports.require (/opt/atom/resources/atom.asar/renderer/api/lib/remote.js:157:34)
    at Atom.module.exports.Atom.confirm (/opt/atom/resources/app/src/atom.js:711:23)
    at Pane.module.exports.Pane.promptToSaveItem (/opt/atom/resources/app/src/pane.js:519:21)
    at Pane.module.exports.Pane.destroyItem (/opt/atom/resources/app/src/pane.js:472:18)
    at HTMLDivElement.<anonymous> (/opt/atom/resources/app/node_modules/tabs/lib/tab-bar-view.js:179:22)
    at space-pen-ul.jQuery.event.dispatch (/opt/atom/resources/app/node_modules/jquery/dist/jquery.js:4430:9)
    at space-pen-ul.elemData.handle (/opt/atom/resources/app/node_modules/jquery/dist/jquery.js:4116:28)
```

### Commands

```
```

### Config

```json
{
  "core": {
    "followSymlinks": true
  }
}
```

### Installed Packages

```coffee
# User
No installed packages

# Dev
No dev packages
```

**Comment by ybounya** [comment]
Duplicate of #6276

**Comment by izuzak** [comment]
Closing in favor of https://github.com/atom/atom/issues/6276. Can you please follow that issue for updates and let us know if when it's fixed you can still reproduce this problem? Thanks for the report.

**Comment by ybounya** [comment]
fixed in 0.190.0
